server/jamovi/core.pyx

```diff
@@ -148,14 +160,18 @@ cdef extern from "columnw.h":
         void setValue[T](int index, T value)
         T value[T](int index)
         const char *getLabel(int value) const
+        const char *getImportValue(int value) const
         int valueForLabel(const char *label) const
+        void appendLevel(int value, const char *label, const char *importValue)
         void appendLevel(int value, const char *label)
```

Comment:
i think you need to remove this. it's not there, and if anyone uses it, it will fail with a cryptic error at link time, rather than a nice error at compile time.